open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -23,9 +23,11 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def column_to_show_by_getting_column_to_diff(
</pre><p>change the function to <code>def append_diff_column(df, new_column, column_to_diff)</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>DONE</p>